Firestore/src/V1/TransactionOptions/ReadOnly.php

```diff
@@ -4,11 +4,6 @@
 
 namespace Google\Cloud\Firestore\V1\TransactionOptions;
 
-/**
- * @deprecated
- */
-class ReadOnly extends PBReadOnly
-{
-}
-
```

Comment:
This is strange, it should not be replacing this file, since we added it to `copy_excludes` in [owlbot.py](https://github.com/googleapis/google-cloud-php/blob/main/Firestore/owlbot.py#L42)

Reply:
I think the difference is that the existing `copy_excludes` are for GAPIC clients, and the `ReadOnly.php` class is coming from Protobuf, so the path will be different.

Reply:
Addressed in https://github.com/googleapis/google-cloud-php/pull/5660

Reply:
Awesome, thanks